libafl_qemu/build_linux.rs

```diff
@@ -26,7 +26,7 @@ pub fn build() {
     } else if cfg!(feature = "systemmode") {
         "systemmode".to_string()
     } else {
-        env::var("EMULATION_MODE").unwrap_or_else(|_| "usermode".to_string())
+        env::var("EMULATION_MODE").unwrap_or_else(|_| "systemmode".to_string())
```

Comment:
This shouldn't be the default I don't think

Reply:
You are right, it won't be in the final PR, it's just for my `rust-analyzer` to not ignore all the `systemmode` code 😅